app/services/migration/migrators/user.rb

```diff
@@ -0,0 +1,34 @@
+module Migration::Migrators
+  class User < Base
+    def call
+      migrate(ecf_users, :user) do |ecf_user|
+        user = ::User.find_or_initialize_by(ecf_id: ecf_user.id)
+
+        user.update!(
+          trn: ecf_user.teacher_profile&.trn ? ecf_user.teacher_profile.trn : ecf_user.npq_applications.map(&:teacher_reference_number).compact.uniq.last,
```

Comment:
should we throw an error if there are multiple TRNs from npq applications 🤔 as there should only be one